### DISCUSSION TITLE
--installer-context System

### DISCUSSION THREAD

**Discussion by DKhalaf** [svrooij/WinTuner#93] [General]
wintuner publish Spotify.Spotify --package-folder C:\Wintuner\Spotify\ --installer-context System

`--installer-context System` does not publish the app to Intune as system. Please advice.

**Comment by svrooij** [comment]
Spotify only has a `User` scope in it's [manifest](https://github.com/microsoft/winget-pkgs/blob/d5c88ebeb1f1721b627387965cd3a2300e750b5c/manifests/s/Spotify/Spotify/1.2.42.290.g242057a2/Spotify.Spotify.installer.yaml)

Are you suggesting that packages that are defined as 'user' might also be installable as `system`? Never tried that.

If you want to test it out, you could manually change the `installerContext` to `1` in the `app.json` file in between the `New-wtWingetPackage` and `Deploy-WtWingetPackage` commands
